plugins/SitesManager/templates/_siteWithoutDataTabs.twig

```diff
@@ -117,6 +153,9 @@
         <h3>{{ 'SitesManager_SiteWithoutDataSinglePageApplication'|translate }}</h3>
         <p>{{ 'SitesManager_SiteWithoutDataSinglePageApplicationDescription'|translate('<a target="_blank" rel="noreferrer noopener" href="https://developer.matomo.org/guides/spa-tracking">','</a>')|raw }}</p>
 
+        <h3>{{ 'SitesManager_SiteWithoutDataCloudflare'|translate}}</h3>
```

Comment:
No need to change it, but the trailing space is missing after `translate`.

Reply:
fixed